docs/src/main/tut/id.md

```diff
@@ -2,8 +2,72 @@
 layout: default
 title:  "Id"
 section: "typeclasses"
-source: "https://github.com/non/cats/blob/master/core/src/main/scala/cats/Id.scala"
-scaladoc: "#cats.Id"
+source: "https://github.com/non/cats/blob/master/core/src/main/scala/cats/package.scala"
+scaladoc: "#cats.Id$"
 ---
 # Id
 
+The identity monad can be seen as the ambient monad that encodes the
+effect of having no effect. It is ambient in the sense that plain pure
+values are values of `Id`.
+
+It is encoded as:
+
+```scala
+type Id[A] = A
+```
+
+That is to say that the type Id[A] is just a synonym for A.  We can
+freely treat values of type `A` as values of type `Id[A]`, and
+vice-versa.
+
+```tut
+import cats.Id._
+
+val x: Id[Int] = 1
+val y: Int = x
+```
+
+Using this type declaration, we can treat our Id type constructor as a
+[Monad](monad.html) and as a [Comonad](comonad.html). The `pure`
+method, which has type `A => Id[A]` just becomes the identity
+function.  The `map` method from `Functor` just becomes function
+application:
+
+```tut
+import cats.Functor
+
+val one: Int = 1
+Functor[Id].map(one)(_ + 1)
+```
+
+When you compare the signatures of `map` and `flatMap` and `coflatMap`:
+
+```scala
+  def map[A, B](fa: Id[A])(f: A => B): Id[B]
+  def flatMap[A, B](fa: Id[A])(f: A => Id[B]): Id[B]
+  def coflatMap[A, B](a: Id[A])(f: Id[A] => B): Id[B]
+```
+
+But then realize that in the flatMap signature, since `Id[B]` is the
```

Comment:
it looks a bit weird to start this sentence with "But then" but it is probably because I am not a native speaker. 

I might have said something like: "when you compare ... you realize that if you replace Id[X] by X, they all have the same signature which is simply function application"